src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -31,8 +33,23 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('api_key')->end()
-                ->scalarNode('api_url')->end()
+                ->scalarNode('api_key')
+                    ->isRequired()
+                    ->info('Gandi api key')
+                ->end()
+                ->scalarNode('api_url')
+                    ->defaultValue(Gandi::DEFAULT_API_URL)
```

Comment:
I don't think it's needed. If null, the client will fallback to this URL.

Reply:
This just more explicit when dumping configuration reference when using Symfony

Reply:
I'm not comfortable to make a const public just to have an example on the configuration.

We should let it private and just add something like `See Gandi class for default one` on the info section.